test/RootCloseWrapperSpec.js

```diff
@@ -100,6 +106,53 @@ describe('RootCloseWrapper', function () {
 
       expect(outerSpy).to.have.not.been.called;
       expect(innerSpy).to.have.been.calledOnce;
+
+      assert.oneOf(innerSpy.getCall(0).args[0].type, ['click', 'mousedown']  );
+    });
+  }
+
+  function shouldCloseOnKeyUp(eventName) {
+    it('should close when escape keyup', () => {
+      let spy = sinon.spy();
+      render(
+        <RootCloseWrapper onRootClose={spy}>
+          <div id='my-div'>hello there</div>
+        </RootCloseWrapper>
+      );
+
+      expect(spy).to.not.have.been.called;
+
+      simulant.fire(document.body, eventName, {keyCode: 27});
```

Comment:
Maybe here too?  Either a local (or even just a comment that explains `27` is the keyCode for escape)?